pwiz_tools/Skyline/TestTutorial/TutorialAuditLogs/en/TestAbsoluteQuantificationTutorial.log

```diff
@@ -31,40 +31,40 @@ Undo Redo : Imported results from 8 files
 Summary   : Imported results from 8 files
 All Info  :
 Imported results from 8 files
-Import results settings > File names : contains "Standard_1.mzML"
```

Comment:
This file shouldn't have to change if you get the mzML use correct based on screenshot mode.